lua/wikis/valorant/MatchPage.lua

```diff
@@ -101,6 +104,171 @@ function MatchPage:populateGames()
 	end)
 end
 
+---@param value number?
+---@param numberOfDecimals number?
+---@return string?
+local function formatNumbers(value, numberOfDecimals)
+	if not value then
+		return nil
+	end
+	numberOfDecimals = numberOfDecimals or 0
+	local format = '%.'.. numberOfDecimals ..'f'
+	return string.format(format, MathUtil.round(value, numberOfDecimals))
+end
+
+-- Total HS% waiting for proper data from crossroads
+---@class ValorantPlayerOverallStats
+---@field acs number[]
+---@field kast number[]
+---@field adr number[]
+---@field hs number[]
+---@field kills number
+---@field deaths number
+---@field assists number
+---@field firstKills number
+---@field firstDeaths number
+---@field totalRoundsPlayed number
+---@field totalKastRounds number
+---@field damageDealt number
+
+---@return Widget?
+function MatchPage:renderOverallStats()
```

Comment:
Can we move the calculation of match aggregated data to the player in the input handler instead?